src/Codacy/Coverage/Parser/CloverParser.php

```diff
@@ -135,4 +135,12 @@ private function getRelativePath(\SimpleXMLElement $fileName)
 
         return $str;
     }
+
+    private function safeDivision($a, $b)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Avoid variables with short names like $a. Configured minimum length is 3.](https://www.codacy.com/app/Codacy/php-codacy-coverage/file/1718512815/issues/source?bid=2159630&fileBranchId=3050296#l139)